hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -497,6 +679,18 @@ fromLedgerTxWitness wits =
   era =
     ShelleyBasedEraAlonzo
 
+--
+-- Value
+--
+
+liftValue :: Value -> TxOutValue Era
+liftValue =
+  TxOutValue MultiAssetInAlonzoEra
+
+fromLedgerValue :: Ledger.Mary.Value Ledger.StandardCrypto -> Value
+fromLedgerValue =
+  fromMaryValue
```

Comment:
Why not rename `fromMaryValue` to `fromLedgerValue`?

Reply:
`fromMaryValue` comes from the `cardano-api` ...